### PR TITLE
[Netatmo] Changed pattern for MAC address in config.xml and new channels for NAModule4

### LINKED ISSUE #2526: [Netatmo]Label changing not working, Module/Device ID wrong
## Current Behavior
I have the Indoor and Outdoor Module and its working. I will Change the Location (paperui) from my outdoor Module but i can't save this new entry. Save button is greyed out the Module and Device ID are wrong, [Screen](https://abload.de/img/netatmo139smi.jpg).

When i changig the Label from my Items i became a Error 405 - Item is not editable. Openhab log saying this:
`2017-08-07 10:56:39.356 [WARN ] [thome.io.rest.core.item.ItemResource] - Cannot update existing item 'netatmo_NAModule1_bc6faa02_020000....e_BatteryVP', because is not managed.
2017-08-07 10:56:53.075 [WARN ] [thome.io.rest.core.item.ItemResource] - Cannot update existing item 'netatmo_NAModule1_bc6faa02_020000...._HeatIndex', because is not managed.
2017-08-07 10:57:03.386 [WARN ] [thome.io.rest.core.item.ItemResource] - Cannot update existing item 'netatmo_NAModule1_bc6faa02_020000...._HeatIndex', because is not managed.
2017-08-07 10:57:32.838 [WARN ] [thome.io.rest.core.item.ItemResource] - Cannot update existing item 'netatmo_NAModule1_bc6faa02_020000...._HeatIndex', because is not managed.
2017-08-07 11:24:44.564 [WARN ] [thome.io.rest.core.item.ItemResource] - Cannot update existing item 'netatmo_NAModule1_bc6faa02_020000...._BatteryVP', because is not managed.
2017-08-07 11:25:10.066 [WARN ] [thome.io.rest.core.item.ItemResource] - Cannot update existing item 'netatmo_NAModule1_bc6faa02_020000...._BatteryVP', because is not managed.`

My Items are added automatically, manually not working because Module/Device ID wrong.
 
## Your Environment

* Version used: OpenHAB2, Addon 2.1.0
* Operating System and version (desktop or mobile, Windows 10, Raspbian Jessie, ...): Raspbian

### PR DESCRIPTION
**Pull request by cweitkamp**
- Changed pattern for MAC address in config.xml (Closes #2526)
- Added channels MinTemp, MaxTemp, DateMinTemp, DateMaxTemp and TempTrend to NAModule4
- Refactored ThingStatus for NetatmoBridgeHandler

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoBridgeHandler.java [hunk 1]
```diff
@@ -65,6 +65,7 @@ public void initialize() {
             getPartnerApi().partnerdevices();
         } catch (RetrofitError e) {
             if (e.getCause() instanceof IOException) {
+                logger.error("Unable to connect Netatmo API : {}", e.getMessage(), e);
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoBridgeHandler.java]: The error is already reported to the thing status, can you lower the log level, especially given:

In [E.4 of the coding guidelines](http://docs.openhab.org/developers/development/guidelines.html#e-logging) we listed what kind of logging level should be used when. Error should only be used:

> `error` logging should only be used to inform the user that something is tremendously wrong in his setup, the system cannot function normally anymore, and there is a need for immediate action. 

Please change this to use the correct logging levels and if possible change the ONLINE/OFFLINE status of thing / bridge.

File: addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoBridgeHandler.java [hunk 2]
```diff
@@ -135,9 +136,13 @@ public PartnerApi getPartnerApi() {
     public NAStationDataBody getStationsDataBody(String equipmentId) {
         if (getStationApi() != null) {
             try {
-                return getStationApi().getstationsdata(equipmentId).getBody();
+                NAStationDataBody data = getStationApi().getstationsdata(equipmentId).getBody();
+                updateStatus(ThingStatus.ONLINE);
+                return data;
             } catch (Exception e) {
-                logger.error("An error occurred while calling station API : {}", e.getMessage());
+                logger.error("An error occurred while calling station API : {}", e.getMessage(), e);
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoBridgeHandler.java]: Maybe lower this one as well.

**Comment by cweitkamp** [comment]
@martinvw Done.

**Comment by martinvw** [comment]
Thanks!

**Pull request merged**